src/styles/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size-cell;
+  max-width: 7 * $width-cell + 6 * $gap-size-cell;
+  padding: 0 10px;
+  margin: 0 auto;
+
+  @for $index from 1 through $month-length {
+    &__day:nth-child(#{$index})::before {
+      content: "#{$index}";
+    }
+  }
+
+  @for $index from 28 to 31 {
```

and also, I would create a variable for 28
```suggestion
  @for $index from 28 to $month-length {
```